keyboards/geekboards/macropad_v2/config.h

```diff
@@ -0,0 +1,78 @@
+/* Copyright 2020 Geekboards ltd. (geekboards.ru / geekboards.de)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x0483
```

Comment:
It looks like this vendor ID belongs STMicroelectronics, and needs to be changed.

https://www.the-sz.com/products/usbid/index.php?v=0x0483&p=&n=

```suggestion
#define VENDOR_ID       0x6770
```
```
echo -n "GeekBoards" | shasum | cut -c1-4 | tr '[:lower:]' '[:upper:]'
6770
```

Reply:
It was officially issued for our device via ST support exclusively for our device. It is guaranteed to be unique, so no problem.

Reply:
verification was PMed to me, so 👍🏻